### ISSUE TITLE
plugin not working on godot 3.3

### ISSUE THREAD

**Issue by titangamingglt** [yalcin-ata/godot-android-plugin-firebase#17]

**Comment by detanaputra** [comment]
is it just me or Godot 3.3 always rewrite all contents on folder [GODOT-PROJECT]/android/build/assets ?
I think it has something to do with that. Since we need to put godot-firebase-config.json file at that folder, when we export the project it got overwritten and because of that, this plugin can not invoke Init method. 

putting godot-firebase-config.json at Godot Project Root did the trick. And now I got another error message 😄

**Comment by kpk-i710** [comment]
> is it just me or Godot 3.3 always rewrite all contents on folder [GODOT-PROJECT]/android/build/assets ?
> I think it has something to do with that. Since we need to put godot-firebase-config.json file at that folder, when we export the project it got overwritten and because of that, this plugin can not invoke Init method.
> 
> putting godot-firebase-config.json at Godot Project Root did the trick. And now I got another error message 😄

yes in 3.3 godot asset folder is overwritten could not fix this problem yet,? otherwise I'm stuck too

**Comment by Aco742** [comment]
[Firebase] >> The value for `apiKey` is not configured. If you are not planning to use it, ignore this message.
[Firebase] >> The value for `authDomain` is not configured. If you are not planning to use it, ignore this message.
[Firebase] >> The value for `databaseURL` is not configured. If you are not planning to use it, ignore this message.
This is the error I am geting when trying to connect to firebase on godot 3.3
This is how my code looks like
func _ready():
	Firebase.Auth.connect("signup_succeeded", self, "_on_register_succeeded")
	


func _on_Register_released():
	var email = $"E-mail".text
	var password = $Password.text
	Firebase.Auth.signup_with_email_and_password(email, password)
	
func _on_register_succeeded(auth_info):
	print(auth_info)

When I was waching tutorial on godot and firebase I saw that I need to setup config.ctf file to set apiKey etc.When I installed 
plugin I didnt get that file so I added it manualy and filled in apiKey and everything from my firebase web/app.So if someone knows how to fix this It would mean a lot to me.Thanks i advance
